skimage/feature/blob.py

```diff
@@ -244,8 +254,24 @@ def blob_dog(image, min_sigma=1, max_sigma=50, sigma_ratio=1.6, threshold=2.0,
     """
     image = img_as_float(image)
 
+    # if both min and max sigma are scalar, function returns only one sigma
+    scalar_sigma = (
+        True if np.isscalar(max_sigma) and np.isscalar(min_sigma) else False
+    )
```

Comment:
This could just be `scalar_sigma = np.isscalar(max_sigma) and np.isscalar(min_sigma)`. =)